datahub-web-react/src/app/entity/user/UserEditProfileModal.tsx

```diff
@@ -0,0 +1,221 @@
+    const [userEmail, setUserEmail] = useState<string | undefined>(data.email);
+    const [userSlack, setUserSlack] = useState<string | undefined>(data.slack);
+    const [userPhoneNumber, setUserPhoneNumber] = useState<string | undefined>(data.phone);
+    const [buttonDisabled, setButtonDisabled] = useState(true);
+
+    // save changes function
+    const onCreateGroup = () => {
+        updateCorpUserPropertiesMutation({
+            variables: {
+                urn: data?.urn || '',
+                input: {
+                    displayName: userName,
+                    title: userTitle,
+                    pictureLink: userImageURL,
+                    teams: userTeam?.split(','),
+                    email: userEmail,
+                    slack: userSlack,
+                    phone: userPhoneNumber,
+                },
+            },
+        })
+            .catch((e) => {
+                message.destroy();
+                message.error({ content: `Failed to Save changes!: \n ${e.message || ''}`, duration: 3 });
+            })
+            .finally(() => {
+                message.success({
+                    content: `Changes saved.`,
+                    duration: 3,
+                });
+                onSave(); // call the refetch function once save
+                // clear the values from edit profile form
+                setUserName('');
+                setUserTitle('');
+                setImageURL('');
+                setUserTeam('');
+                setUserEmail('');
+                setUserSlack('');
+                setUserPhoneNumber('');
+            });
+        onClose();
+        onSave();
+    };
+
+    // userFieldValidation before submitting the changes.
+    const userFieldValidations = () => {
+        if (buttonDisabled) return true;
+        return false;
+    };
+
+    return (
+        <Modal
+            title="Edit Profile"
+            visible={visible}
+            onCancel={onClose}
+            footer={
+                <>
+                    <Button onClick={onClose} type="text">
+                        Cancel
+                    </Button>
+                    <Button onClick={onCreateGroup} disabled={userFieldValidations()}>
+                        Save Changes
+                    </Button>
+                </>
+            }
+        >
+            <Form
+                form={form}
+                initialValues={{ ...data }}
+                autoComplete="off"
+                layout="vertical"
+                onFieldsChange={() => setButtonDisabled(form.getFieldsError().some((field) => field.errors.length > 0))}
+            >
+                <Form.Item
+                    name="name"
+                    label={<Typography.Text strong>Name</Typography.Text>}
+                    rules={[
+                        {
+                            required: true,
+                            message: 'Please enter the User name.',
```

Comment:
Change the message to: "Enter a display name"

Reply:
Required changes are added.